Monika After Story/game/script-topics.rpy

```diff
@@ -10476,6 +10476,64 @@ label monika_yellowwp:
 
     return "derandom"
 
+# have you read amontillado
+default persistent._mas_pm_read_cask_of_amontillado = None
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_amontillado",
+            category=['literature'],
+            prompt="The Cask of Amontillado",
+            unlocked=True,
```

Comment:
Perhaps you added a needless ```,``` at the end of this line?

Reply:
That was because of the commented line 10490 which is now taking its place, thanks for pointing it out